src/frontend/src/lib/components/contact/Avatar.svelte

```diff
@@ -38,7 +66,7 @@
 	</div>
 {:else}
 	<span
-		class={`${commonClasses} inline-block inline-flex items-center justify-center bg-[lightgray] font-bold text-white ${styleClass}`}
+		class={`${commonClasses} inline-block inline-flex items-center justify-center font-bold text-white transition-colors duration-[1s] ${styleClass}`}
```

Comment:
nitpick: is the same?

```suggestion
		class={`${commonClasses} inline-block inline-flex items-center justify-center font-bold text-white transition-colors duration-1000 ${styleClass}`}
```

Reply:
Yes, prefer 1000, I'll change.